idp-service/src/main/resources/amr_acr_mapping.json

```diff
@@ -1,28 +1,14 @@
 {
-  "amr_values" : {
-    "only_pin" :  [{ "name": "pin" }],
-    "only_otp" :  [{ "name": "otp" }],
-    "only_finger" :  [{ "name": "fpt", "count": 1, "bioSubTypes" : ["leftThumb"] }],
-    "only_iris" :  [{ "name": "iris", "count": 2 }],
-    "five_fingers" : [{ "name": "fpt", "count": 4 , "bioSubTypes" : ["unknown", "unknown", "unknown", "unknown"]}],
-    "otp_one_finger" : [{ "name": "otp" },{ "name": "fpt", "count": 1 , "bioSubTypes" : ["rightThumb"]}],
-    "otp_all_fingers" : [{ "name": "otp" },{ "name": "fpt", "count": 10 }],
-    "iris_otp" :  [{ "name": "iris", "count": 2 }, { "name": "otp" }]
-  },
-  "acr_values" : {
-    "level0" : "0",
-    "level1" : "1",
-    "level2" : "2",
-    "level3" : "3",
-    "level4" : "4",
-    "level5" : "5"
+  "amr" : {
+    "PIN" :  [{ "type": "PIN" }],
+    "OTP" :  [{ "type": "OTP" }],
+    "Inji" :  [{ "type": "INJI" }],
+    "L1 device" :  [{ "type": "BIO", "count": 1 }]
```

Comment:
If user uses iris and if we give 1, will it create issue ?

Reply:
As per our MOSIP SBI Spec, iris modality with counts 2 and 1 are valid.